src/Loop.php

```diff
@@ -178,11 +178,18 @@ public static function repeat($interval, callable $callback, $data = null)
     /**
      * Execute a callback when a stream resource becomes readable or is closed for reading.
      *
+     * Warning: Closing resources locally, e.g. with `fclose`, might not invoke the callback. Be sure to `cancel` the
+     * watcher when closing the resource locally. Drivers might choose to notify the user in a debug mode if there are
```

Comment:
s/might choose to notify the user in a debug mode/may choose to notify the user

There's no need for talking about a debug mode - that's an implementation detail unrelated to the specification here.